apps/meteor/ee/app/livechat-enterprise/server/lib/LivechatEnterprise.js

```diff
@@ -10,7 +10,7 @@ import { removeUserFromRoles } from '../../../../../server/lib/roles/removeUserF
 import {
 	processWaitingQueue,
 	removePriorityFromRooms,
-	updateInquiryQueuePriority,
+	updateInquiryQueueSla,
 	updatePriorityInquiries,
 	updateRoomPriorityHistory,
```

Comment:
Are these also from "priorities" or just named similar?

Reply:
good catch, didn't get those, it's priorities, yes